### PR TITLE
change bloxstaking image to ssv

### PR DESCRIPTION
**Pull request by guym-blox**

### CODE REVIEW

**Review by Tom-ssvlabs — APPROVED** [review]

**Review by Tom-ssvlabs — APPROVED** [review]

**Review by Tom-ssvlabs — APPROVED** [review]

File: docs/OPERATOR_GETTING_STARTED.md [hunk 1]
```diff
@@ -152,20 +152,20 @@ Before start, make sure the clock is synced with NTP servers.
...
 ```shell
-$ docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode version
+$ docker run --rm -it 'ssvlabs/ssv-node:latest' /go/bin/ssvnode version
 ```
 
-In order to update, kill running container and pull the latest image or a specific version (`bloxstaking/ssv-node:<version>`)
+In order to update, kill running container and pull the latest image or a specific version (`ssvlabs/ssv-node:<version>`)
 ```shell
-$ docker rm -f ssv_node && docker pull bloxstaking/ssv-node:latest
+$ docker rm -f ssv_node && docker pull ssvlabs/ssv-node:latest
```
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:168]: sorry I mislead you, lets revert this back to bloxstaking..

File: docs/bootnode.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ _Note: This is an example. Replace the placeholders as explained below._
    docker rm -f ssv_bootnode && docker run -d --restart unless-stopped --name=ssv_bootnode \
        -e CONFIG_PATH=/config.yaml -p 5000:5000 -p 4000:4000/udp \
        -v $(pwd)/config.yaml:/config.yaml -v $(pwd):/data -it \
-       'bloxstaking/ssv-node-unstable:latest' make BUILD_PATH=/go/bin/ssvnode start-boot-node
+       'ssvlabs/ssv-node-unstable:latest' make BUILD_PATH=/go/bin/ssvnode start-boot-node
```
Comment (y0sher) [inline docs/bootnode.md:57]:
```suggestion
       'bloxstaking/ssv-node-unstable:latest' make BUILD_PATH=/go/bin/ssvnode start-boot-node
```

File: scripts/generate_local_config.sh [hunk 3]
```diff
@@ -19,7 +19,7 @@ function create_operators() {
   mkdir -p config
 
   for ((i=1;i<=OP_SIZE;i++)); do
-    docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode generate-operator-keys > tmp.log
+    docker run --rm -it 'ssvlabs/ssv-node:latest' /go/bin/ssvnode generate-operator-keys > tmp.log
```
Comment (y0sher) [inline scripts/generate_local_config.sh:22]:
```suggestion
    docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode generate-operator-keys > tmp.log
```

File: docs/OPERATOR_GETTING_STARTED.md [hunk 4]
```diff
@@ -152,20 +152,20 @@ Before start, make sure the clock is synced with NTP servers.
 Then, run the docker image in the same folder you created the `config.yaml`:
 
 ```shell
-$ docker run -d --restart unless-stopped --name=ssv_node -e CONFIG_PATH=./config.yaml -p 13001:13001 -p 12001:12001/udp -v $(pwd)/config.yaml:/config.yaml -v $(pwd):/data --log-opt max-size=500m --log-opt max-file=10 -it 'bloxstaking/ssv-node:latest' make BUILD_PATH=/go/bin/ssvnode start-node \
+$ docker run -d --restart unless-stopped --name=ssv_node -e CONFIG_PATH=./config.yaml -p 13001:13001 -p 12001:12001/udp -v $(pwd)/config.yaml:/config.yaml -v $(pwd):/data --log-opt max-size=500m --log-opt max-file=10 -it 'ssvlabs/ssv-node:latest' make BUILD_PATH=/go/bin/ssvnode start-node \
```
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:155]:
```suggestion
$ docker run -d --restart unless-stopped --name=ssv_node -e CONFIG_PATH=./config.yaml -p 13001:13001 -p 12001:12001/udp -v $(pwd)/config.yaml:/config.yaml -v $(pwd):/data --log-opt max-size=500m --log-opt max-file=10 -it 'bloxstaking/ssv-node:latest' make BUILD_PATH=/go/bin/ssvnode start-node \
```

File: docs/OPERATOR_GETTING_STARTED.md [hunk 5]
```diff
@@ -79,7 +79,7 @@ $ ./install.sh
 The following command will generate your operator's public and private keys (appear as "pk" and "sk" in the output). 
 
 ```
-$ docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode generate-operator-keys
+$ docker run --rm -it 'ssvlabs/ssv-node:latest' /go/bin/ssvnode generate-operator-keys
```
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:82]:
```suggestion
$ docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode generate-operator-keys
```

File: README.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ to run validators in a decentralized and trustless way.
 ## Information
 
 ### General SSV information (Semi technical read)
-* Article by [Blox](https://medium.com/bloxstaking/an-introduction-to-secret-shared-validators-ssv-for-ethereum-2-0-faf49efcabee)
+* Article by [ssv](https://medium.com/bloxstaking/an-introduction-to-secret-shared-validators-ssv-for-ethereum-2-0-faf49efcabee)
```
Comment (y0sher) [inline README.md:32]:
```suggestion
* Article by [ssv](https://blog.ssv.network/an-introduction-to-secret-shared-validators-ssv-for-ethereum-2-0-faf49efcabee)
```

**Review by moshe-blox — DISMISSED** [review]

File: docs/OPERATOR_GETTING_STARTED.md [hunk 7]
```diff
@@ -152,20 +152,20 @@ Before start, make sure the clock is synced with NTP servers.
...
+$ docker run -d --restart unless-stopped --name=ssv_node -e CONFIG_PATH=./config.yaml -p 13001:13001 -p 12001:12001/udp -v $(pwd)/config.yaml:/config.yaml -v $(pwd):/data --log-opt max-size=500m --log-opt max-file=10 -it 'ssvlabs/ssv-node:latest' make BUILD_PATH=/go/bin/ssvnode start-node \
   && docker logs ssv_node --follow
 ```
 
 ### 7. Update SSV Node Image
 
 The current version is available through logs or a cmd:
 ```shell
-$ docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode version
+$ docker run --rm -it 'ssvlabs/ssv-node:latest' /go/bin/ssvnode version
```
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:163]:
```suggestion
$ docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode version
```

File: docs/OPERATOR_GETTING_STARTED.md [hunk 8]
```diff
@@ -152,20 +152,20 @@ Before start, make sure the clock is synced with NTP servers.
...
 ```
 
-In order to update, kill running container and pull the latest image or a specific version (`bloxstaking/ssv-node:<version>`)
+In order to update, kill running container and pull the latest image or a specific version (`ssvlabs/ssv-node:<version>`)
```
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:166]:
```suggestion
In order to update, kill running container and pull the latest image or a specific version (`bloxstaking/ssv-node:<version>`)
```

File: docs/OPERATOR_GETTING_STARTED.md (hunk 1 above)
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:168]:
```suggestion
$ docker rm -f ssv_node && docker pull bloxstaking/ssv-node:latest
```

File: docs/OPERATOR_GETTING_STARTED.md (hunk 4 above)
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:155]:
```suggestion
$ docker run -d --restart unless-stopped --name=ssv_node -e CONFIG_PATH=./config.yaml -p 13001:13001 -p 12001:12001/udp -v $(pwd)/config.yaml:/config.yaml -v $(pwd):/data --log-opt max-size=500m --log-opt max-file=10 -it 'bloxstaking/ssv-node:latest' make BUILD_PATH=/go/bin/ssvnode start-node \
```

File: docs/OPERATOR_GETTING_STARTED.md (hunk 5 above)
Comment (y0sher) [inline docs/OPERATOR_GETTING_STARTED.md:82]:
```suggestion
$ docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode generate-operator-keys
```

**Review by y0sher — APPROVED** [review]

**Review by Tom-ssvlabs — APPROVED** [review]

**Review by moshe-blox — APPROVED** [review]

**Pull request merged**